### PR TITLE
Chore: bootstrap our fork from pegjs-import

### PR DESCRIPTION
**Pull request by rodrigo4244**

### CODE REVIEW

**Comment by rodrigo4244** [comment]
The circleCI is failing because there isn't any merged PR. So, `assert-version-bump` is not going to work.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 
+Copyright (c) 2014 Harry Schmidt
```
Comment (gunar) [inline LICENSE]: ?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
-# pegjs-import
+# @invisible/pegjs-import
 
-[![Build Status](https://travis-ci.org/casetext/pegjs-import.svg?branch=master)](https://travis-ci.org/casetext/pegjs-import)
+[![CircleCI](https://circleci.com/gh/invisible-tech/pegjs-imp/tree/master.svg?style=svg)](https://circleci.com/gh/invisible-tech/pegjs-imp/tree/master)
```
Comment (gunar) [inline README.md]: I will rename this repo `pegjs-imp` to `pegjs-import` then this badge will break. Please update.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 
 Import other PEG files into a PEG.js grammar.
 
+Forked from [pegjs-import](https://github.com/casetext/pegjs-import).
```
Comment (gunar) [inline README.md]:
```md
Forked from [casetext](https://github.com/casetext/pegjs-import).

File: package.json [hunk 4]
```diff
@@ -1,10 +1,11 @@
 {
   "name": "@invisible/pegjs-import",
-  "version": "0.2.7",
+  "version": "1.0.0",
```
Comment (gunar) [inline package.json]: Why are you increasing a major version?

File: package.json [hunk 5]
```diff
@@ -18,13 +19,14 @@
...
     "chai": "^3.5.0",
     "mocha": "^3.0.2"
   },
   "author": "Harry Schmidt",
   "license": "MIT",
   "repository": {
-    "type" : "git",
-    "url" : "http://github.com/casetext/pegjs-import.git"
+    "type": "git",
+    "url": "http://github.com/invisible-tech/pegjs-imp.git"
```
Comment (gunar) [inline package.json]: I will rename this repo so update this line.

File: circle.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+test:
+  override:
+    - yarn test
+
+deployment:
+  release:
+    branch: master
+    commands:
+      - ./node_modules/.bin/publish
```
Comment (gunar) [inline circle.yml]: Use our publish package

File: LICENSE (hunk 1 above)
Comment (rodrigo4244) [reply LICENSE]: We should use our license isn't it? Now, thinking straight, it makes more sense.

File: package.json (hunk 4 above)
Comment (rodrigo4244) [reply package.json]: Because it is our release. I thought that we should reset the version and start with `1.0.0`.

File: circle.yml (hunk 6 above)
Comment (rodrigo4244) [reply circle.yml]: But this is the bin of our `publish` package.

**Pull request merged**